tests/integration/cattletest/core/test_allocation.py

```diff
@@ -229,7 +232,8 @@ def test_host_vnet_association(admin_client, sim_context,
 
 
 def test_allocation_stay_associated_to_host(admin_client, sim_context):
-    c = admin_client.create_container(imageUuid=sim_context['imageUuid'])
+    c = admin_client.create_container(
```

Comment:
Ah ok, I wont call them all out then. Give the whole PR a quick scan for whitespace only changes and revert them.